docker-compose.yml

```diff
@@ -1,10 +1,13 @@
-postgres:
-  image: sameersbn/postgresql
-  ports:
-  - "5432:5432"
-  environment:
-    - DB_USER=developer
-    - DB_PASS=developer
-    - DB_NAME=db_test,db_dev
-  volumes:
-    - ./data/postgres:/var/lib/postgresql/data
+version: '3'
+
+services:
+  postgres:
+    image: postgres
+    ports:
+    - "5432:5432"
+    environment:
+      - DB_USER=developer
```

Comment:
Should we use this as default credentials because it is development, or use .env variables for example?
for example: DB_USER=${DB_USER}

Don't know if this has extra use for development purposes?